de.dlr.sc.virsat.model.extension.requirements.test/.project

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>de.dlr.sc.virsat.model.extension.requirements.test</name>
+	<comment></comment>

```

Comment:
Not sure where to put the: The .project is fine but the plugin project itself is missing an about.html file.

Reply:
Seems like mylyn settings are missing here in this plugin as well. Have not seen it in the .settings folder.